libs/wingc/src/type_check.rs

```diff
@@ -273,6 +273,17 @@ pub struct Namespace {
 	// the types after initial compilation.
 	#[derivative(Debug = "ignore")]
 	pub loaded: bool,
+
+	/// Where we can resolve this namespace from
+	pub module_path: ResolveSource,
+}
+
+#[derive(Debug)]
+pub enum ResolveSource {
+	/// A wing file within the source tree for this compilation
+	WingFile,
+	/// Something like an external JSII module. This will will the spec for the module.
```

Comment:
>  This will will the spec for the module.

??

Reply:
idk what you mean, makes perfect sense

(jk, will fix)